hackathon_site/review/test_admin.py

```diff
@@ -267,6 +270,23 @@ def test_permission_denied(self):
         response = self.client.get(self.change_view)
         self.assertEqual(response.status_code, status.HTTP_403_FORBIDDEN)
 
+    def test_reviewer_group_correct_permissions(self):
+        """
+        Ensure the reviewer group has all the permissions it is supposed to and nothing else.
```

Comment:
You should actually hard-code the expected permissions into this test. This test currently checks that the things in the variable are set, but not that they are the ones that we want.

This also goes along with my other comment. Migrations are run during tests, so if someone were to remove something from that array, the migration would update accordingly (bad) and this test would still pass (also bad).

Reply:
Since the permissions should be hard-coded into the migrations and test, you don't really need the `REVIEWER_PERMISSIONS` variable at all.